Find a functionally equivalent code:com.android.internal.telephony.CallerInfo.doSecondaryLookupIfNecessary:COMMENT
Method Modifier: default     static      internal    
Comment:/**
 * Performs another lookup if previous lookup fails and it's a SIP call
 * and the peer's username is all numeric. Look up the username as it
 * could be a PSTN number in the contact database.
 *
 * @param context the query context
 * @param number the original phone number, could be a SIP URI
 * @param previousResult the result of previous lookup
 * @return previousResult if it's not the case
 */

@@ -2,8 +2,8 @@
     if (!previousResult.contactExists && PhoneNumberUtils.isUriNumber(number)) {
         String username = PhoneNumberUtils.getUsernameFromUriNumber(number);
         if (PhoneNumberUtils.isGlobalPhoneNumber(username)) {
-            previousResult = getCallerInfo(context, Uri.withAppendedPath(PhoneLookup.CONTENT_FILTER_URI, Uri.encode(username)));
+            previousResult = getCallerInfo(context, Uri.withAppendedPath(PhoneLookup.ENTERPRISE_CONTENT_FILTER_URI, Uri.encode(username)));
         }
     }
     return previousResult;

